platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -423,20 +430,14 @@ public class SwirldsPlatform implements Platform {
 
             // When we perform the migration to using birth round bounding, we will need to read
```

Comment:
this comment should move as well

Reply:
I know what you mean now, based on clarifying comments on slack. This is actually intentional. I want this comment to be next to the PCES logic, which did not move.